signup/signup.js

```diff
@@ -589,14 +601,28 @@ $('#vitaSignupForm').submit(function(e) {
 		}
 	});
 
+	var dependents = [];
+	$(".dependent-div").each(function() { 
+		var firstName = $(this).find('.firstName').val().trim();
+		var lastName = $(this).find('.lastName').val().trim();
+
+		if (firstName.length > 0 && lastName.length > 0) { // empty strings
```

Comment:
So if either the first or last name field is left empty, then it just simply won't add the dependent to the database and the client will have no way of knowing that the dependent they wanted to add did not get added.

I think it would be a better UX for those fields to both be required. If they try to submit the form without filling out one of them, it will throw an on screen error and allow them to go fix it. If they don't want to add the dependent, then they would just need to hit the remove button.

Reply:
Good call, will change now